modules/atomic-widgets/elements/atomic-svg/atomic-svg.php

```diff
@@ -60,22 +63,31 @@ protected function define_atomic_controls(): array {
 	}
 
 	protected function define_base_styles(): array {
-		$width = Size_Prop_Type::generate( [
+		$display_value = String_Prop_Type::generate( 'inline-block' );
```

Comment:
The `a` tag by default uses `display: inline;` which doesn't respect the sizes that we define in the styles.